network-api/networkapi/wagtailpages/models.py

```diff
@@ -911,7 +911,6 @@ def get_context(self, request):
         # We need to expose MEDIA_URL so that the s3 images will show up properly
         # due to our custom image upload approach pre-wagtail
         context = super(Homepage, self).get_context(request)
-        print(settings.MEDIA_URL)
```

Comment:
spotted this while figuring out why MEDIA_URL doesn't include the AWS directory, so... remove'd